### ISSUE TITLE
Goto Next Error or Warning doesn't show up in F1 anymore

### ISSUE THREAD

**Issue by dbaeumer** [microsoft/vscode#10335]
- VSCode Version: Insider

![capture](https://cloud.githubusercontent.com/assets/1931590/17509684/89dc7702-5e1b-11e6-8808-02645b6230c3.png)

@alexandrudima assigning to you since you do work in that area right now and the action looks ok for me.

**Comment by alexandrudima** [comment]
@dbaeumer This works for me on master:

The precondition for the actions is `EditorContextKeys.Writable`. Are you trying this in a read-only editor?

![image](https://cloud.githubusercontent.com/assets/5047891/17510173/0c0f3dca-5e1e-11e6-9019-4f4f295daf2e.png)

**Comment by dbaeumer** [comment]
@alexandrudima no readonly editor. Seeing this in the following build with the following file open:

![capture](https://cloud.githubusercontent.com/assets/1931590/17510611/00d0dde0-5e20-11e6-9a9d-7f0832ee7fcc.png)

**Comment by alexandrudima** [comment]
@dbaeumer Agreed. I can reproduce in the built version of yesterday. Today's builds apparently failed.

**Comment by alexandrudima** [comment]
Verified that the latest insiders build shows the two actions in the F1 list
